cob_bringup/components/ros_control_base.launch

```diff
@@ -7,21 +7,23 @@
 	<arg name="can_device" default="can0"/>
 	<arg name="pkg_hardware_config" default="$(find cob_hardware_config)"/>
 	<arg name="sim" default="false"/>
+	<arg name="reset_errors_before_recover" default="false"/>
 
 	<arg name="component_name" value="base"/>
 	<arg name="interval_ms" value="20"/>
 	<arg name="use_external_sync" value="false"/>
 	<arg name="start_external_sync" value="false"/>
 
+	<rosparam unless="$(arg sim)" ns="$(arg component_name)/driver" command="load" file="$(arg driver_yaml)"/>
```

Comment:
This is the right place, I don't know why it was below beforehand.
It might break other robots with outdated configs..